felix/bpf/bpf.go

```diff
@@ -387,6 +387,69 @@ type mapEntry struct {
 	Err   string   `json:"error"`
 }
 
+func (me *mapEntry) UnmarshalJSON(data []byte) error {
+	type entry struct {
+		Key   []string    `json:"key"`
+		Value interface{} `json:"value"`
+		Err   string      `json:"error"`
+	}
+
+	if string(data) == "null" {
+		return nil
+	}
+
+	var e entry
+	err := json.Unmarshal(data, &e)
+	if err != nil {
+		return nil
```

Comment:
Why suppress this one?

Reply:
we cannot skip over some entries, so a poor mans solution saying, the entry does not look like we expect so just return an empty entry which we later on filter out. But I guess I will go with proper parsing as you suggest above.